en/02_Developer_Guides/06_Testing/index.md

```diff
@@ -3,23 +3,19 @@ title: Testing
 summary: Deploy robust applications by bundling Unit and Behavior tests with your application code and modules.
 ---
 
-# Unit and Integration Testing
-
-For behaviour testing in Silverstripe CMS, check out [Silverstripe CMS Behat Documentation](https://github.com/silverstripe-labs/silverstripe-behat-extension/).
```

Comment:
I'll add it back in here and link to the doc page (such as it is) - I removed that because I was going to do _all_ the changes in a single PR (and as part of it, add docs about behat in here properly rather than linking off to some module).

Reply:
Actually nevermind, I won't.

This heading basically says "This documentation is only about unit and functional testing, go elsewhere for behat docs - but there _is_ a page here for behavioural testing already (even if it only points elsewhere itself) - so this heading and link is not needed.